Description: Disable some consistently failing gio tests
Author: Marc Deslauriers <marc.deslauriers@canonical.com>
Forwarded: not-needed

--- a/gio/tests/meson.build
+++ b/gio/tests/meson.build
@@ -93,13 +93,6 @@ gio_tests = {
   'g-icon' : {},
   'gdbus-addresses' : {},
   'gdbus-message' : {},
-  'gdbus-peer' : {
-    'dependencies' : [libgdbus_example_objectmanager_dep],
-    'install_rpath' : installed_tests_execdir,
-    # FIXME: https://gitlab.gnome.org/GNOME/glib/-/issues/1392
-    # FIXME: https://gitlab.gnome.org/GNOME/glib/-/issues/3148
-    'can_fail' : host_system in ['darwin', 'windows', 'gnu'],
-  },
   'inet-address' : {},
   'io-stream' : {},
   'max-version' : {'install' : false},
@@ -143,12 +136,6 @@ gio_tests = {
   'tls-database' : {'extra_sources' : ['gtesttlsbackend.c']},
   'tls-bindings' : {'extra_sources' : ['gtesttlsbackend.c']},
   'unix-fd' : {},
-  'gdbus-address-get-session' : {
-    'extra_programs': host_system != 'windows' ? ['dbus-launch'] : [],
-    # FIXME: https://gitlab.gnome.org/GNOME/glib/-/issues/1392
-    # FIXME: https://gitlab.gnome.org/GNOME/glib/-/issues/3148
-    'can_fail' : host_system in ['darwin', 'gnu'],
-  },
   'win32-appinfo' : {},
 }
 
